test/system_test.py

```diff
@@ -131,6 +131,18 @@ def spack_env_dev_install_and_test(spack_env: str,
         f'git clone --depth 1 --recurse-submodules -b {icon_branch} git@github.com:C2SM/icon.git {unique_folder}',
         check=True,
         shell=True)
+
```

Comment:
We should not patch ICON.
I will prepare a branch with +grib2 in ICON which will need to be merged first

Reply:
ok , after discussion this is a solution considering the that this is the C2SM verison of ICON which uses the environement in the CI. Please inform C2SM before merging